assets/css/style.css

```
@@ -132,12 +113,12 @@ header {
   padding-top: 70px;
 }
 
-.ingredients-item-2 {
-margin: 0px 30px 0px 30px
+.ingredients-item-spice {
+margin: 0px 30px;
 }
 
 .ingredients-divider {
-  padding: 25px 0px 25px 0px;
+  padding: 25px 0px;
```

```suggestion
  padding: 25px 0;
```
and better to use 
padding-top
padding-bottom
for such cases